src/main/resources/assets/api-docs.yaml

```diff
@@ -2959,110 +3032,149 @@ definitions:
       referenceId:
         type: string
         description: The reference id for this DAR
-      investigator:
-        type: string
-        description: 'Describes the name of the Principal Investigator.'
-      institution:
-        type: string
-        description: 'Describes the name of Institution.'
-      department:
-        type: string
-        description: 'Describes the name of the Institution Department.'
-      division:
-        type: string
-        description: 'Describes the name of the Division.'
-      address1:
+      userId:
+        type: integer
+        format: int32
+        description: Describes the id of the User (Researcher).
+      createDate:
         type: string
-        description: 'Describes the principal address.'
-      address2:
+        description: Creation Date
+      sortDate:
         type: string
-        description: 'Describes the secondary address. Optional.'
-      city:
+        description: Sort Date
```

Comment:
Another nitpick, I'd like to have a more expanded description for the dates in the docs. Jonathan and I didn't know what exactly some of the dates represented in our last retro meeting and I expect similar problems in the future. Having it laid out in the docs would be a good way to provide context

Reply:
It doesn't have to be longer than one or two sentences

Reply:
I'm also waffling on the format. I think I'll change them to longs so the UI can work with them more easily.

Reply:
Done, and changed dates in json to number.